sqlite_fdw.c

```diff
@@ -1572,14 +1573,36 @@ make_tuple_from_result_row(sqlite3_stmt * stmt,
 		int			attnum = lfirst_int(lc) - 1;
 		Oid			pgtype = TupleDescAttr(tupleDescriptor, attnum)->atttypid;
 		int32		pgtypmod = TupleDescAttr(tupleDescriptor, attnum)->atttypmod;
+		//List	   *options = NULL;
+		int			sqlite_value_affinity;
 
-		if (sqlite3_column_type(stmt, attid) != SQLITE_NULL)
+		sqlite_value_affinity = sqlite3_column_type(stmt, stmt_colid);
+		if ( sqlite_value_affinity != SQLITE_NULL)
 		{
-			is_null[attnum] = false;
-			row[attnum] = sqlite_convert_to_pg(pgtype, pgtypmod,
-											   stmt, attid, festate->attinmeta, attnum);
+		    /* Here will be processing of column options about special convert behaviour 
+		    options = GetForeignColumnOptions(rel, attnum_base);
```

Comment:
Is this code unnecessary? If so, please remove it.

Reply:
This code will be uncommented during implementation of [data transformation table](https://github.com/mkgrgis/sqlite_fdw/tree/readme-fix-and-additions#datatypes). @t-kataym have wrote this implementation need to change SQL behaviour during column and other levelled options. It's important to leave this necessary draft  for other contributors. My next PR will be about PostgreSQL encoding, PR after next about human readable error message (uncommenting) and than first PR with [data transformation table](https://github.com/mkgrgis/sqlite_fdw/tree/readme-fix-and-additions#datatypes) implementation and introducing a new option or options.

Reply:
I don't want to break any tests with the message in this PR, but will uncomment in near future.

Reply:
Thank for your explanation,

However, we cannot accept the PR containing such comment code.
Leaving commented-out code in source code is bad practice, as it takes up space, causes confusion, and leads to maintenance issues if it's not removed.

Could you remove it or change it to a TODO note?

Reply:
Yes, I have reduced this draft to one TODO comment line. It's really "leaving commented-out code in source code is bad practice".

Reply:
Could you check the other place also?

Reply:
Yes, please comment on lines after my [new commit](https://github.com/pgspider/sqlite_fdw/pull/74/commits/90d9064a4195be51214dd34088541a202a64a9af) with missing code style. Now I have no idea where is problems.